modules/seqexec/model/src/main/scala/seqexec/model/TelescopeGuideConfig.scala

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2016-2019 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package seqexec.model
+
+import cats.Eq
+import cats.implicits._
+import monocle.macros.Lenses
+import seqexec.model.enum.MountGuideOption
+import seqexec.model.enum.M1GuideConfig
+import seqexec.model.enum.M2GuideConfig
+
+/** Data type for guide config. */
+@Lenses
+final case class TelescopeGuideConfig(mountGuide: MountGuideOption,
+                                      m1Guide:    M1GuideConfig,
+                                      m2Guide:    M2GuideConfig)
+
+@SuppressWarnings(Array("org.wartremover.warts.PublicInference"))
```

Comment:
Given #967 I suppose it doesn't matter but out of curiosity where is the public inference that is being suppressed here?

Reply:
`WartRemover` and the `@Lenses` don't like each other because of reasons. So we need to suppress it to let `@Lenses` to work. One of the main reasons I disilke `WartRemover`

Reply:
I guess the code it writes doesn't have an explicit type.  Seems like they should, but it's irrelevant now.